android/app/src/main/java/com/ddangddangddang/android/feature/messageRoom/MessageRoomActivity.kt

```diff
@@ -31,6 +32,15 @@ class MessageRoomActivity :
     lateinit var messageAdapter: MessageAdapter
 
     private val adapter by lazy { ConcatAdapter(roomCreatedNotifyAdapter, messageAdapter) }
+
+    private val messageReceiver: MessageReceiver by lazy {
```

Comment:
이 메시지 리시버는 register하면 MessageRoomActivity가 살아있는 동안에만 동작하는건지 궁금합니다!

Reply:
![image](https://github.com/woowacourse-teams/2023-3-ddang/assets/67176829/6c0826c3-e8b6-4ea2-bc77-6a044287016f)

since the lifetime of such a
     * registered BroadcastReceiver is tied to the object that registered it.</p>
 코드 주석 중 일부인데, 아마 런타임에 등록한 브로드 캐스트 리시버는 이 녀석의 생명주기에 묶이는 것 같습니다.
 
 unregister-가 있기는한데 단순히 수신 더이상 하기 싫을 때 사용하는 것 같습니다.
 따로 예제 코드들을 봐도 해제를 넣었거나, 반드시 넣어야 한다는 문장은 못봤습니다. 
 회전 했을때 터지지 않고 잘 동작했습니다.
 
가장 핵심은 사실, 브로드 캐스트 리시버에는 수신 외에 생명주기 함수가 없어서, 딱히 자체적으로 살아있을 것 같지는 않습니다. 
![image](https://github.com/woowacourse-teams/2023-3-ddang/assets/67176829/976c30ee-eef7-453d-8b90-fee3edb7b637)

Reply:
와 해제 해줘야 하네요. 테스트해보니까 안해주면, 또 등록돼서 두 번 수신되네요

Reply:
매번 방에 들어갈때마다 등록될 것 같은데 이 친구가 액티비티가 죽으면 같이 죽는 친구인지 궁금했어요!
액티비티에 종속되는 친구는 아닌가보네요🤔🤔🤔